src/App.tsx

```diff
@@ -2,108 +2,46 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+
+import { Post } from './types/Post';
+import { PostFromServer } from './types/PostFromServer';
+import { CommentFromServer } from './types/CommentFromServer';
+import { UserFromServer } from './types/UserFromServer';
+import { PostList } from './components/PostList';
+
+function getUserById(
+  post: PostFromServer,
+  users: UserFromServer[],
```

Comment:
hard to read, also you can use usersFromServer here without passing it as a parameter because you have it in the same file